examples/platforms/nrf52840/README.md

```diff
@@ -53,17 +65,17 @@ The driver can be found in `third_party/NordicSemiconductor/libraries/usb/nordic
 
 You can build the libraries with support for USB bootloader with automatic USB DFU trigger support in PCA10059. As this dongle uses the native USB support, you must enable it as well.
 
-To build the libraries, run make with the following parameter:
+To build the libraries, run make with the following parameters:
 
 ```
 $ make -f examples/Makefile-nrf52840 USB=1 BOOTLOADER=1
 ```
 
 See [nRF52840 Dongle Programming][nrf52840-dongle-programming] for more details about how to use the USB bootloader.
 
-[nrf52840-dongle-programming]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52%2Fdita%2Fnrf52%2Fdevelopment%2Fnrf52840_dongle%2Fprogramming.html&cp=2_0_4_4
+[nrf52840-dongle-programming]: https://www.nordicsemi.com/DocLib/Content/User_Guides/nrf52840_dongle/latest/UG/nrf52840_Dongle/programming
 
-### Native SPI Slave support
+### Native SPI support
 
 You can build the libraries with support for native SPI Slave.
 To build the libraries, run make with the following parameter:
```

Comment:
Edits for lines 94-96:

In this example, [spi-hdlc-adapter][spi-hdlc-adapter] is a tool that you can use to communicate between NCP and wpantund over SPI. For this example, `spi-hdlc-adapter` is installed in `/usr/bin`.